From: Tommy Webb <tommy@calyxinstitute.org>
Date: Thu, 6 Jun 2024 15:55:16 +0000
Subject: Temp build fix for "Bromite subresource adblocker"

"INFO" not found for some reason, even when including base/logging.h

Change-Id: Ie39abcda8ef855ef2063cead3d4c9233258bb75c
---
 .../subresource_filter/core/browser/verified_ruleset_dealer.cc  | 2 --
 1 file changed, 2 deletions(-)

diff --git a/components/subresource_filter/core/browser/verified_ruleset_dealer.cc b/components/subresource_filter/core/browser/verified_ruleset_dealer.cc
--- a/components/subresource_filter/core/browser/verified_ruleset_dealer.cc
+++ b/components/subresource_filter/core/browser/verified_ruleset_dealer.cc
@@ -40,8 +40,6 @@ RulesetFilePtr VerifiedRulesetDealer::OpenAndSetRulesetFile(
                "VerifiedRulesetDealer::OpenAndSetRulesetFile", "file_valid",
                file->IsValid());
 
-  LOG(INFO) << "OpenAndSetRulesetFile: " << file_path << " is valid: " << file->IsValid();
-
   if (file->IsValid()) {
     SetRulesetFile(file->Duplicate());
     expected_checksum_ = expected_checksum;
-- 

